common/helpers/field/flatten-conditional-fields.js

```diff
@@ -0,0 +1,27 @@
+const { isObject, concat } = require('lodash')
+
+function flattenConditionalFields([key, field]) {
+  if (!field.items) {
+    return [key, field]
+  }
+
+  return [
+    key,
+    {
+      ...field,
+      items: field.items.map(item => {
```

Comment:
My fave hobby horse -  I think doing the items assignment before the return statement would make it easier to understand both what's happening to the items and also what's being returned